### PR TITLE
Feature/rtl toolbar

### PR DESCRIPTION
**Pull request by khaykov**
This PR fixes toolbar in RTL direction.

[https://gyazo.com/a388afe138a6960ebc43b21e1a945cba](https://gyazo.com/a388afe138a6960ebc43b21e1a945cba)

To test:

1. Change locale to RTL language or toggle "Force RTL layout direction" in Dev options.
2. Check that basic and advanced toolbars work in RTL mode.

Note: Toolbar is using `HorizontalScrollView` which did not get  RTL support until API 19. On API 17 and 18 LTR layout will be forced.

### CODE REVIEW

File: aztec/src/main/kotlin/org/wordpress/aztec/toolbar/AztecToolbar.kt [hunk 1]
```diff
@@ -702,43 +709,54 @@ class AztecToolbar : FrameLayout, OnMenuItemClickListener {
...
+                        //keep hard focus on media button to avoid it
+                        toolbarScrolView.requestChildFocus(buttonMediaCollapsed, buttonMediaCollapsed)
+                        layoutExpanded.startAnimation(layoutExpandedTranslateInEnd)
                     }
                 }
         )
     }
 
+    fun scrollToBeginingOfToolbar() {
+        if (TextUtilsCompat.getLayoutDirectionFromLocale(Locale.getDefault()) == ViewCompat.LAYOUT_DIRECTION_LTR
```
Comment (malinajirka) [inline aztec/src/main/kotlin/org/wordpress/aztec/toolbar/AztecToolbar.kt]: We might consider adding a comment explaining that HorizontalScrollView on older android versions doesn't support RtL layouts.

File: aztec/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -1,5 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 
-<manifest package="org.wordpress.aztec">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+          package="org.wordpress.aztec">
+
+    <application
```
[the remark this answers is not in the document]
Comment (khaykov) [reply aztec/src/main/AndroidManifest.xml]: Thanks for explaining - I wasn't too sure about this myself :)  I moved the attribute to the app, which makes way more sense.

File: aztec/src/main/kotlin/org/wordpress/aztec/toolbar/AztecToolbar.kt (hunk 1 above)
Comment (khaykov) [reply aztec/src/main/kotlin/org/wordpress/aztec/toolbar/AztecToolbar.kt]: Good call! I added a comment about RTL support to this method and to the toolbar itself.

**Comment by khaykov** [comment]
Ready for another round 👍

**Comment by malinajirka** [comment]
Thanks! ;)

**Pull request merged**